vagrant.yml

```diff
@@ -12,6 +12,8 @@ synced_folders:
     # If using the 'smb2' type above the user and share will be configured automatically.
     smb_username: 'vagrant'
     smb_password: 'vagrant'
+    # Create smb share on vagrant up. Set to false if you already have smb shared folder.
```

Comment:
Please change to "Create smb share on vagrant up. Set to false if you want to or already created the user and the smb share manually."

Also, it is not obvious what the share name should be in order for this to work. So worth adding a note here as well.